packages/installer/upgrade.sh

```diff
@@ -160,19 +160,24 @@ upgrade() {
     export LUET_PRIVILEGED_EXTRACT=true
 
     args=""
-    if [ -z "$VERIFY" ]; then
-        args="--enable-logfile --logfile /tmp/luet.log --plugin image-mtree-check"
+    if [ -z "$VERIFY" ] || [ "$VERIFY" == true ]; then
```

Comment:
Is there something that sets this var to true?

Reply:
it could be set to true if someone sets it in the `/etc/cos-upgrade-image` file, that file gets sourced to be able to set custom behavior of `cos-upgrade` by default